test/fops/t.c

```diff
@@ -78,8 +81,9 @@ main(int argc, char *argv[])
 	nops = 1000;
 	nthreads = 10;
 	runs = 1;
+	explicit_transaction = false;
 	config_open = working_dir = NULL;
-	while ((ch = __wt_getopt(progname, argc, argv, "C:h:l:n:r:t:")) != EOF)
+	while ((ch = __wt_getopt(progname, argc, argv, "C:h:l:n:r:t:i")) != EOF)
```

Comment:
I suggest `-x` or `-T` for using txns.  If you use `T` then it should sort alphabetically before `t:` in the `wt_getopt` arg list.  If you use `x` then it sorts where it currently is.  I have no intuitive connection between using txns and `-i`.